test/addons/openssl-client-cert-engine/binding.gyp

```diff
@@ -0,0 +1,10 @@
+{
+  'targets': [
+    {
+      'target_name': 'testengine',
```

Comment:
You may want to use `'product_extension': 'engine',` or something like this here, and replace `.so` with `.engine`.